metadata_tools/hosts/GO_0xxx/GO_0xxx_geometry_cloud.py

```
@@ -0,0 +1,74 @@
+                   volumes=[task_data['volume_id']])
+
+    return False, None
+
+#========================================================================================
+async def main():
+    # These command line arguments are used to override environment variables when
+    # specifying the behavior of the worker process manager. They are optional
+    # and most useful when running the worker locally.
+
+    # parse metadata arguments
+    host, index_type, template_dir = util.parse_template_name(hconf.template_name)
```

Ruff (0.14.10)

Unpacked variable `index_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)